src/components/dashboard/components/dictionary/common/DictionaryModal.jsx

```diff
@@ -338,7 +332,8 @@ export class DictionaryModal extends React.Component {
                   </FormGroup>
                   <FormGroup style={{ marginTop: '12px' }}>
                     Description{' '}
-                    <FormControl
+                    <Input
+                      type="textarea"
                       componentClass="textarea"
```

Comment:
@Duncanian Remove `componentClass` prop, `type`prop is used in reactstrap instead and you used it.

Reply:
I have done it.